src/js/popup/popup.js

```diff
@@ -1240,10 +1251,15 @@
 
         if (serverStoragePref) {
           try {
-            return await browser.runtime.sendMessage({
+            const masks = await browser.runtime.sendMessage({
               method: "getAliasesFromServer",
               options,
             });
+
+            await browser.storage.local.set({"relayAddresses": masks});
+
+            return masks;
+
```

Comment:
The crux of the PR. Whenever we query `getAliasesFromServer`, we store it locally as well. 

TODO: Move this logic to background script so it happens anytime it happens from anywhere.

Reply:
So...we already do this. 

https://github.com/mozilla/fx-private-relay-add-on/blob/main/src/js/background/background.js#L60

Removed in 7fa9c62db3928021dc827f160cdaabda3bfed543